package.json

```diff
@@ -24,6 +24,8 @@
   "author": "Mozilla (https://mozilla.org/)",
   "license": "MPL-2.0",
   "dependencies": {
+    "babel-middleware": "git://github.com/shane-tomlinson/babel-middleware.git#45c06180",
```

Comment:
On Fri, Jul 15, 2016 at 2:30 PM, Vlad Filippov <notifications@github.com>
wrote:

> In package.json
> <https://github.com/mozilla/fxa-content-server/pull/3841#discussion_r70972728>
> :
>
> > @@ -24,6 +24,8 @@
> >    "author": "Mozilla (https://mozilla.org/)",
> >    "license": "MPL-2.0",
> >    "dependencies": {
> > +    "babel-middleware": "git://github.com/shane-tomlinson/babel-middleware.git#45c06180",
>
> @shane-tomlinson <https://github.com/shane-tomlinson> seems like you had
> do a lot of fixes and the module is not maintained that much anymore, is it
> possible to republish it with a different name and give the rest of the
> team access to the fork / npm publishing?
>

Exactly what I want to do!